runtime/common/src/evm/precompile.rs

```diff
@@ -76,21 +82,52 @@ where
 	}
 
 	fn is_precompile(&self, address: H160) -> bool {
-		[
-			ECRECOVER_ADDR,
-			SHA256_ADDR,
-			RIPEMD160_ADDR,
-			IDENTITY_ADDR,
-			MODEXP_ADDR,
-			BN128ADD_ADDR,
-			BN128MUL_ADDR,
-			BN128PAIRING_ADDR,
-			BLAKE2F_ADDR,
-			SHA3FIPS256_ADDR,
-			DISPATCH_ADDR,
-			ECRECOVERPUBLICKEY_ADDR,
-		]
-		.contains(&address.0)
+		matches!(
+			address.0,
+			ECRECOVER_ADDR
+				| SHA256_ADDR | RIPEMD160_ADDR
+				| IDENTITY_ADDR | MODEXP_ADDR
+				| BN128ADD_ADDR | BN128MUL_ADDR
+				| BN128PAIRING_ADDR
+				| BLAKE2F_ADDR | SHA3FIPS256_ADDR
+				| DISPATCH_ADDR | ECRECOVERPUBLICKEY_ADDR
+		)
+	}
+}
+
+/// A set of precompiles. This set might contain
+/// not yet mainnet ready precompiles in order to test
+/// those in development or staging environment without touching
+/// the mainnet set.
+pub struct Development<R>(CentrifugePrecompiles<R>);
```

Comment:
@mustermeiszer how do you envision merging with with https://github.com/centrifuge/centrifuge-chain/pull/1490? like do want to add a custom `Altair` precompiles struct a la this `Development` or how do you see us doing that?

Reply:
I would add an Altair type maybe? Whatever you find cleaner